samples/cancelJob.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(jobId) {
+  // [START bigquery_cancel_job]
+  // Import the Google Cloud client library
+  const {BigQuery} = require('@google-cloud/bigquery');
+
+  async function cancelJob(jobId = 'existing-job-id') {
```

Comment:
Instead of accepting the `jobId` as a parameter here, we should rely on the parameter getting passed into the `main` method above.  Then we should have one of those:

```js
// TODO(developer): uncomment the following lines
// const jobId = 'existing-job-id';
```

... type of thingies.

Reply:
@alixhami had a point that people miss the commented out variables so it's easier to see what to change with the input. I really have no opinion either way, just want to know if there is some standard so I can be consistent.

Reply:
Ah, understood.  The usage of that pattern is codified here:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head

If we want to change it - it's something we'd have to all talk about, agree on, plan a large scale change, etc.  I want to be really careful about following the stuff in the rubric.

Reply:
It's a bit confusing between languages, because the rubric committee's recommendation was default parameters, not commented out variables. Then, some languages deviated from that once the per-language sample standard pages were created.

Reply:
Ah, so the good news here is that we do both!  Please do keep the default parameters, just move em up to the `main` function.  

The default parameters are for folks who clone the repo and run the samples locally.  The commented out declarations are for folks browsing cloud.google.com.  Those folks won't see the `main` method in the snippet ... so commented out we go :)

Reply:
(side note - would be a great topic to start an internal thread on so we can all get on the same page)

Reply:
Thanks both!!